samples/subsys/usb/console/src/main.c

```diff
@@ -6,33 +6,26 @@
 
 #include <zephyr.h>
 #include <sys/printk.h>
-#include <sys/util.h>
-#include <string.h>
 #include <usb/usb_device.h>
 #include <drivers/uart.h>
 
+BUILD_ASSERT(DT_NODE_HAS_COMPAT(DT_CHOSEN(zephyr_console), zephyr_cdc_acm_uart),
+	     "Console device is not ACM CDC UART device");
+
 void main(void)
 {
-	const struct device *dev = device_get_binding(
-			CONFIG_UART_CONSOLE_ON_DEV_NAME);
+	const struct device *dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_console));
```

Comment:
It is excellent that this sample can be reworked in this way with the latest update.